src/backend/marsha/core/tests/test_api_liveregistration.py

```diff
@@ -0,0 +1,779 @@
+"""Tests for the LiveRegistration API."""
+from datetime import timedelta
+import json
+import random
+import uuid
+
+from django.test import TestCase
+from django.utils import timezone
+
+from rest_framework_simplejwt.tokens import AccessToken
+
+from ..factories import (
+    ConsumerSiteFactory,
+    LiveRegistrationFactory,
+    PlaylistFactory,
+    VideoFactory,
+)
+from ..models import LiveRegistration
+
+
+class LiveRegistrationApiTest(TestCase):
```

Comment:
I think at least 2 tests are missing : 

```
def test_api_liveregistration_read_detail_token_good_email_no_context_id_wrong_context_id(self):
        """The video is portable is an other playlist portable from an other consumer site, the registration should not be return."""
        consumer_site = ConsumerSiteFactory()
        video = VideoFactory(playlist__consumer_site=consumer_site)
        
        other_consumer_site = ConsumerSiteFactory()
        other_playlist = PlaylistFactory(consumer_site=other_consumer_site)

        liveregistration = LiveRegistrationFactory(
            email="sarah@openfun.fr",
            video=video,
        )

        jwt_token = AccessToken()
        jwt_token.payload["resource_id"] = str(video.id)
        jwt_token.payload["context_id"] = str(other_playlist.lti_id)
        jwt_token.payload["user"] = {
            "email": "sarah@openfun.fr",
            "id": "56255f3807599c377bf0e5bf072359fd",
            "username": "John",
        }

        response = self.client.get(
            f"/api/liveregistrations/{liveregistration.id}/",
            HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
        )

        self.assertEqual(response.status_code, 404)

    def test_api_liveregistration_read_detail_token_good_email_wrong_context_id(self):
        """The video is portable is an other playlist portable from an other consumer site, the registration should not be return."""
        consumer_site = ConsumerSiteFactory()
        video = VideoFactory(playlist__consumer_site=consumer_site)
        
        other_consumer_site = ConsumerSiteFactory()
        other_playlist = PlaylistFactory(consumer_site=other_consumer_site)

        liveregistration = LiveRegistrationFactory(
            email="sarah@openfun.fr",
            video=video,
            lti_user_id="56255f3807599c377bf0e5bf072359fd",
            lti_consumer_site=video.playlist.consumer_site,
        )

        jwt_token = AccessToken()
        jwt_token.payload["resource_id"] = str(video.id)
        jwt_token.payload["context_id"] = str(other_playlist.lti_id)
        jwt_token.payload["user"] = {
            "email": "sarah@openfun.fr",
            "id": "56255f3807599c377bf0e5bf072359fd",
            "username": "John",
        }

        response = self.client.get(
            f"/api/liveregistrations/{liveregistration.id}/",
            HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
        )

        self.assertEqual(response.status_code, 404)
```

Reply:
As long as we have an email in the token, this information is only needed to access the registration, no need to compare with consumer_site or user_id in this case, so these two extra tests will be 200 and not 404.
All it matters to us is that this email has access to this video
Maybe we should speak about it.